appsmithorg/appsmith
# fix: Update copy and flag for all toggle buttons on Admin settings page

## Description

> Updated the copy and flags for all toggle buttons on the Admin settings page so it is easy to understand on the UI.

Fixes #11702 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Go to the Admin Settings page
- Click on the Authentication tab.
- Click edit for Form Login authentication method.
- Check the default toggle value for Form login should be Enabled and the flag should be on.
- On clicking the toggle and saving the changes, the form login should turn off (provided some other authentication method is enabled).

[![LOOM DEMO](http://cdn.loom.com/sessions/thumbnails/cdead02a72fb44979829b1fadce2f96a-00001.gif)](https://www.loom.com/share/cdead02a72fb44979829b1fadce2f96a)

## Checklist:






## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>


    // Code coverage diff between base branch:release and head branch: hotfix/toggle-buttons 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.91 **(0)** | 37.28 **(0.01)** | 35.3 **(0)** | 56.24 **(0)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>
